server/tools/docker-entrypoint.sh

```diff
@@ -38,6 +39,9 @@ file_env 'KEYCLOAK_PASSWORD'
 
 if [[ -n ${KEYCLOAK_USER:-} && -n ${KEYCLOAK_PASSWORD:-} ]]; then
     /opt/jboss/keycloak/bin/add-user-keycloak.sh --user "$KEYCLOAK_USER" --password "$KEYCLOAK_PASSWORD"
+else
```

Comment:
You can drop this ```else``` branch and replace the empty string initialization in the previous ```if``` as follows (IOW simplify the whole if/else e.g. as):

```
KEYCLOAK_USER=${KEYCLOAK_USER:-$DEFAULT_USER}
KEYCLOAK_PASSWORD=${KEYCLOAK_PASSWORD:-$DEFAULT_PASSWORD}


if [[ -n ${KEYCLOAK_USER} && -n ${KEYCLOAK_PASSWORD} ]]; then
    /opt/jboss/keycloak/bin/add-user-keycloak.sh --user "$KEYCLOAK_USER" --password "$KEYCLOAK_PASSWORD"
```

This way, one one call to `add-user-keycloak.sh` would cover adding admin credentials in both cases (the specified ones, or the default ones) IIUC.